Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/Competition/BlueLeftAutonomousCompetition.java

```diff
@@ -0,0 +1,53 @@
+package org.firstinspires.ftc.team6220_PowerPlay.Competition;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.team6220_PowerPlay.AprilTagDetect;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+@Autonomous(name = "BlueLeft", group = "Competition")
+public class BlueLeftAutonomousCompetition extends AprilTagDetect {
+
+    @Override
+    public void runOpMode() throws InterruptedException {
+        initialize();
+
+        servoGrabber.setPosition(0.45);
```

Comment:
Shouldn't you use your `driveGrabber` method for this?

Reply:
This is a different position just for autonomous